src/mcp.ts

```
@@ -16,7 +16,7 @@ import {
 	loadSessionBlockData,
 	loadSessionData,
 } from './data-loader.ts';
-import { dateSchema } from './types.internal.ts';
+import { dateSchema } from './_types.ts';
```

ESLint

[error] Expected "./_types.ts" to come before "./data-loader.ts".

(perfectionist/sort-imports)